docs/api/api/api.txt

```diff
@@ -1933,6 +1933,15 @@ XmlResult: required_checks
 == Staging
   WARNING: This API is currently in beta and unstable
 
+=== Staging Workflow
+
+<staging_workflow_project>: The project for which we want to create the staging workflow
+
+POST /staging/<staging_workflow_project>/workflow
+
+XmlBody: managers_group
```

Comment:
That's not a good API. The next setting required would mean a complete new root node.

Reply:
I am not sure what you mean. What would make it a good API?

Reply:
Well, 
```
<staging_workflow>
  <manager_group>test-staging</manager_group>
</staging_workflow>
```

Having the one setting you require *right now* in the root node doesn't allow any extension later.

Reply:
app/views/models/staging/_workflow.xml.builder uses `<workflow/>`

Reply:
I get it, thanks! :cupid: